part-08.mdx

```diff
@@ -0,0 +1,230 @@
+# Build your first custom dashboard
+
+In previous parts of the tutorial, you have learned how several sections of Netdata dashboard worked. 
+
+This guide will show you how to easily create custom pages and set up a custom dashboard to fit your custom 
+or specific needs.
+
+## What you'll learn in this part
+
+In this part of the Netdata guide, you'll learn how to:
+
+-   create a custom-dashboard.html file
+-   Add dashboard.js
+-   Set the default Netdata server
+-   Create a chart, set its duration and size
+-   Set dimensions for your custom chart
+
+Let's get on with it!
+
+## Create a custom-dashboard.html file
+
+You can create your dashboard using simple HTML; in some cases for basic dashboards, 
+you may not need Javascript.
+
+On the same custom dashboard, you can make use of any available chart libraries ([Google Chart](https://developers.google.com/chart/), [D3.js](http://d3js.org/), and so on).
+
+You can also use data from one or any number of Netdata servers on the same dashboard as well as 
+host your dashboard HTML page on any web server.
+
+You can also add Netdata charts to existing web pages.
+
+In your local Netdata installation (http://myhost:19999/dashboard.html), you can find a custom dashboard template 
+which houses samples of all supported charts that can help you get a good idea of how to put together your custom dashboard.
+
+On your local server, `/usr/share/netdata/web` is the default web root directory with the HTML and JS files.
+
+It is worthy of note that the main dashboard is also in that directory and called `index.html`. 
+The `index.html` has a different syntax which you should not use as a template for custom dashboards.
+
+To create a new dashboard, add the following on an empty html file:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Your dashboard</title>
+
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="apple-mobile-web-app-capable" content="yes">
+  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
+
+  <!-- here you will add dashboard.js -->
+
+</head>
+<body>
+
+<!-- here you will add charts -->
+
+</body>
+</html>
+```
+
+## Add dashboard.js
+
+You need to include the `dashboard.js` file of a Netdata server to add Netdata charts to any web page.
+
+For example, if your Netdata server listens at `http://LOCAL_IP:19999/`, 
```

Comment:
Before, you reference a server running at `http://myhost:19999/dashboard.html`, and now `LOCAL_IP` here. Maybe you can make them consistent? I don't really care which way you go, but the change might confuse people.